src/app/components/core/scheduler/list-schedules/list-schedules.component.ts

```diff
@@ -49,7 +49,7 @@ export class ListSchedulesComponent implements OnInit {
            *
            */
           data.schedules.forEach(sch => {
-            if (!['south_c', 'north_c', 'north_C', 'south', 'north', 'notification_c', 'automation_script'].includes(sch.processName)) {
+            if (!['south_c', 'north_c', 'north_C', 'south', 'north', 'notification_c', 'automation_script', 'dispatcher_c', 'bucket_storage_c'].includes(sch.processName)) {
```

Comment:
and management?

note. Management has 2 schedules. We will have to think about polling interval schedule, for now it can remain here.